src/misc/sphere-collider.js

```diff
@@ -117,11 +121,16 @@ module.exports = {
         mesh.geometry.computeBoundingSphere();
         radius = mesh.geometry.boundingSphere.radius;
         distance = position.distanceTo(meshPosition);
-        if (distance < radius + data.radius) {
+        scale = scaleFactor(mesh.getWorldScale(meshScale));
+        if (distance < radius * scale + colliderRadius) {
           collisions.push(el);
           distanceMap.set(el, distance);
         }
       }
+      // use max of scale factors to maintain bounding sphere collision
+      function scaleFactor (scaleVec) {
+        return Math.max.apply(null, Object.values(scaleVec));
```

Comment:
`Object.values` is not available in Safari — I would use `scaleVec.toArray()` here.